index.html

```diff
@@ -1167,15 +1170,14 @@ <h3>Getting Started</h3>
         </pre>
 
         <p>
-Once a developer has prototyped their credential to a point where they believe
-all of the credential properties are stable, it is advised that they generate
-vocabulary and context files for their application and publish them at stable
-URLs, so that other developers can use the same vocabulary and context to achieve
-interoperability. The `https://www.w3.org/ns/credentials/examples/v2` URL above
+After stabilizing all credential properties, developers are advised to generate
+and publish vocabulary and context files at stable URLs to facilitate
+interoperability with other developers. The 
+`https://www.w3.org/ns/credentials/examples/v2` URL above
 would then be replaced with the URL of a use-case-specific context. This
 process is covered in Section [[[#extensibility]]]. Alternatively,
 developers can reuse existing vocabulary and context files that happen to fit
-their use case. They can explore the [[[VC-SPECS]]] [[VC-SPECS]] for reusable
+their use-case. They can explore the [[[VC-SPECS]]] [[VC-SPECS]] for reusable
```

Comment:
```suggestion
their use case. They can explore the [[[VC-SPECS]]] [[VC-SPECS]] for reusable
```

The word "use case" is not hyphenated.